_layouts/base.html

```diff
@@ -24,6 +24,7 @@
   <link rel="stylesheet" href="assets/css/jquery.fancybox.css" type="text/css" media="all" />
   <link rel="stylesheet" href="assets/css/style-green.css" id="mainstyle" type="text/css" media="all" />
   <link rel="stylesheet" href="assets/css/custom.css" id="custom" type="text/css" media="all" />
+  {% for css_name in page.extra_css %}<link rel="stylesheet" href="/assets/css/{{ css_name }}">{% endfor %}
```

Comment:
Just like the JS: No need to special case the CSS includes, we should just load them all in `base.html`